src/eip7594/cell.c

```diff
@@ -26,7 +26,7 @@
  */
 void print_cell(const Cell *cell) {
     for (size_t i = 0; i < FIELD_ELEMENTS_PER_CELL; i++) {
-        const Bytes32 *field = (const Bytes32 *)&cell->bytes[i * BYTES_PER_FIELD_ELEMENT];
-        print_bytes32(field);
+        const Bytes32 *field_element = (const Bytes32 *)&cell->bytes[i * BYTES_PER_FIELD_ELEMENT];
```

Comment:
Using something like `element_fr` would be slightly more consistent with the rest of the code.
But it's a bit ugly.

Reply:
Hmm, it would need to be `element_bytes` in this case, as that's the type. I'll do that 👍